x/incentives/simulation/params.go

```diff
@@ -17,10 +17,10 @@ const (
 
 // TODO: remove hardcoded params
 // refer x/epochs/simulation/genesis.go.
+// TODO: x/epochs/simulation/genesis.go doesn't exist, what was this comment trying to say?
```

Comment:
x/epochs/simulation/genesis.go doesn't exist, what was this comment trying to say?

Reply:
I think it encourages randomizing the choice of `epocIdentifiers`. Dev is working on reworking the simulator so I don't believe this TODO is actionable. We can just remove it IMO